odoo/tools/config.py

```diff
@@ -599,9 +599,9 @@ def load(self):
             p.read([self.rcfile])
             for (name,value) in p.items('options'):
                 name = outdated_options_map.get(name, name)
-                if value=='True' or value=='true':
+                if value in ('True', 'true', 'T', 't', 1, '1'):
```

Comment:
I think we should consider TRUE and FALSE too.
We can simplify the conditions by converting value to string and then using `lower`.

Reply:
something like this?
```suggestion
                if type(value) == str and value.lower in ("t", "true") or value in (1, "1"):
```

Reply:
or cast to string beforehand, then just check in `("t", "true", "1")`

Reply:
Late to the party, but there is also `odoo.tools.misc.str2bool` https://github.com/unipartdigital/odoo/blob/9da977c16237b3bd21638da9aa8a5fd39bcd40ca/odoo/tools/misc.py#L470